pkg/json/ingest.go

```diff
@@ -98,17 +99,21 @@ func (ingester *Ingester) ingest(input jsonom.Node, path ls.NodePath, schemaNode
 		}
 		return node, dp, nil
 	}
-
-	if schemaNode != nil && schemaNode.GetTypes().Has(ls.AttributeTypes.Polymorphic) {
-		return validate(ingester.ingestPolymorphicNode(input, path, schemaNode))
-	}
-	switch next := input.(type) {
-	case *jsonom.Object:
-		return validate(ingester.ingestObject(next, path, schemaNode))
-	case *jsonom.Array:
-		return validate(ingester.ingestArray(next, path, schemaNode))
+	// only ingest nodes that have a matching schema attribute
+	if ingester.OnlySchemaAttributes {
```

Comment:
It should ingest if schemaNode is not nil, or if OnlySchemaAttributes is not set

if schemaNode!=nil || !ingester.OnlySchemaAttributes